CRM/Eck/BAO/EckEntityType.php

```diff
@@ -19,25 +19,22 @@
 
 class CRM_Eck_BAO_EckEntityType extends CRM_Eck_DAO_EckEntityType {
 
-  protected static $_entityTypes;
-
-  public static function getEntityTypes() {
-    if (!isset(static::$_entityTypes)) {
-      $entity_types = civicrm_api3('EckEntityType', 'get', [], ['limit' => 0])['values'];
-      static::$_entityTypes = array_combine(
-        array_map(
-          function ($name) {
-            return 'Eck' . $name;
-          },
-          array_column($entity_types, 'name')
-        ),
-        $entity_types
-      );
```

Comment:
The previous version of this function used a verbose `array_map` to index the array with `'Eck' . $name`, however I did a usage search and found that the array keys are never accessed! So I removed it.